db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.14
--- Dumped by pg_dump version 9.5.14
+-- Dumped from database version 9.5.16
+-- Dumped by pg_dump version 9.5.16
```

Comment:
What generated this change?

Reply:
I think `bundle exec rake db:migrate`

Reply:
You don't need to run db:migrate, this PR is not adding any migration.

db:setup db:seed

Should be enough

Reply:
In some point @zarpilla sync this branch with develop and we have this migration pending https://github.com/coopdevs/timeoverflow/commit/9f7b99a794f2934d67b2b0f3c91533060eccd973#diff-91c81d30c0bc5a60d6b1397f04e0570e

Reply:
Better to `db:drop` and `db: setup db:seed` again in those cases.

Reply:
Hi, a couple of questions,

Should I drop and recreate the database again or is it ok now?
The drop and setup won't modify the structure.sql file?

Reply:
I would checkout from `develop` the files that you don't need to touch in your PR. For instance `structure.sql`.

Then `db:drop` and `db:setup db:seed`.

Reply:
Done, the file has been reverted to "develop" version